WebService/WebService/Data/DbAgent.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Comparison_Engine.Base_Classes;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebService.Base_Classes;
+
+namespace WebService.Data
+{
+    public class DbAgent
+    {
+        //private string connectionString = "Server=tcp:comparison-engine.database.windows.net,1433;Initial Catalog=Comparison Engine;Persist Security Info=False;User ID=PSIadmin;Password=CompEngine123Ė;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;";
+        public static List<Bar> GetBars(DbContextOptions<DBContext> options)
+        {
+            try
+            {
+                using (var dbContext = new DBContext(options))
+                {
+                    return dbContext.Bars.ToList();
+                }
+            }
+            catch (Exception e)
+            {
+                
+                throw;
+            }
```

Comment:
empty whitespace :D where's the convention that we agreed on, Lebowski???
But for real, we need to start thinking about logging exceptions and not just throwing them left and right

Reply:
Left space for exception handling for later.